data/compatibility/supportStatus.yml

```diff
@@ -10,9 +10,9 @@
 - version: "1.26"
   supported: "Yes"
   releaseDate: "May 08, 2025"
-  eolDate: "~Dec 2025 (Expected)"
-  k8sVersions: ["1.29", "1.30", "1.31", "1.32"]
-  testedK8sVersions: ["1.24", "1.25", "1.26", "1.27", "1.28"]
+  eolDate: "~Sept 2025 (Expected)"
+  k8sVersions: ["1.30", "1.31", "1.32", "1.33"]
```

Comment:
Sorry... lower these... I haven't figured out how to bump what the tests are doing and haven't gotten a reply yet. Tests are only on 1.32 and they won't change in time for release.